### ISSUE TITLE
Using an untrained model, equivalent to the pre-trained model

### ISSUE THREAD

**Issue by miraodasilva** [facebookresearch/av_hubert#53]
TLDR: How do I load an untrained AV-HuBERT model?

Hi,

Congratulations on this very successful project and thanks for sharing the code. I experimented now with loading the pre-trained model, as explained in the README, and it works well. I would like to load an equivalent, untrained model, to check the difference. There are instructions on running commands to train a model from scratch, but I simply want to load the model itself, as everything else will be done in my code. So, is there an equivalent to this line 

`models, cfg, task = fairseq.checkpoint_utils.load_model_ensemble_and_task([ckpt_path])`

to load this same model configuration, but untrained (so without loading the checkpoint)? Apologies in advance for my unfamiliarity with your codebase and fairseq as a whole.

Thanks,
Rodrigo

**Comment by chevalierNoir** [comment]
Hi,

Setting `model.no_pretrained_weights` to `true` will set up the model architecture but not load pre-trained weights. See [here](https://github.com/facebookresearch/av_hubert/blob/main/avhubert/hubert_asr.py#L41-L44).

**Comment by miraodasilva** [comment]
I see, thank you!!
